sdk/metrics/src/main/java/io/opentelemetry/sdk/metrics/aggregation/Aggregations.java

```diff
@@ -186,15 +215,21 @@ public AggregatorFactory getAggregatorFactory(InstrumentValueType instrumentValu
           : DoubleLastValueAggregator.getFactory();
     }
 
+    @Override
+    public Accumulation merge(Accumulation a1, Accumulation a2) {
+      // TODO: Define the order between accumulation.
```

Comment:
Do we need a spec issue for this, or just a Java issue?

Reply:
I think right now go uses a timestamp as well associated with the entry. I left a TODO because I intend to look into this soon.